kie-docs/shared/Workbench/Authoring/DataSets/DataSets.xml

```diff
@@ -0,0 +1,728 @@
+        It's up to the user to decide on the best approach.
+      </para>
+
+      <para>
+        Depending on the case might be better to define the filter at a data set level for reusing across other modules. The decision may also have impact on the performance
+        since a filtered cached data set will have far better performance than a lot of individual non-cached data set lookup requests. (See the next section for more information about
+        caching data sets).
+      </para>
+
+      <note>
+        <para>
+          Notice, for SQL data sets, the user can use both the filter feature introduced or, alternatively, just add custom filter criteria to the SQL sentence. Although, the first
+          approach is more appropriated for non technical users since they might not have do the required SQL language skills.
+        </para>
+      </note>
+    </section>
+
+  </section>
+
+  <!-- Data set edition -->
+
+  <section xml:id="wb.DataSetDefEditor">
+    <title>Data set editor</title>
+
+    <para>
+      To edit an existing data set definition go the data set explorer, expand the desired data set definition and click on the <emphasis>Edit</emphasis> button.
+      This will cause a new editor panel to be opened and placed on the center of the screen, as shown in the next screenshot:
+    </para>
+
+    <figure>
+      <title>Data set definition editor</title>
+      <screenshot>
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="images/Workbench/Authoring/DataSets/DataSetDefEditor.png" format="PNG"/>
+          </imageobject>
+        </mediaobject>
+      </screenshot>
+    </figure>
+
+    Every time we edit an item its editor is added to the center panel. We can navigate through the list of opened editors just by clicking on the down arrow icon placed at the
+    editor's toolbar in the top right corner.
+
+    <figure>
+      <title>Editor selector</title>
+      <screenshot>
+        <mediaobject>
+          <imageobject>
+            <imagedata fileref="images/Workbench/Authoring/DataSets/DataSetDefEditorSelector.png" format="PNG"/>
+          </imageobject>
+        </mediaobject>
+      </screenshot>
+    </figure>
+
+    The editor provides all the features described in previous sections. We can change the configuration settings, test our data set definition and modify the resulting data set
+    structure. Additionally, the editor provides some extra buttons in its toolbar:
+
+    <itemizedlist>
+      <listitem>
+        <para>
+          Save - To validate the current changes and store the data set definition.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Delete - To remove it permanently from the data set definition storage. Any client module referencing the data set may be affected.
+        </para>
+      </listitem>
+      <listitem>
+        <para>
+          Validate - To check if the current configuration is correct.
```

Comment:
It would be nice to add 1 or two sentences about a typical cases where the validation might fail. Bad db connection parameters? Or wrong format of DS Definition? Can user create invalid DS Def directly in the editor?